examples/react/react-app/src/App.tsx

```diff
@@ -1,33 +1,47 @@
-import { useComponentValue } from "@dojoengine/react";
-import { Entity } from "@dojoengine/recs";
 import { useEffect, useState } from "react";
 import "./App.css";
-import { Direction } from "./utils";
-import { getEntityIdFromKeys } from "@dojoengine/utils";
 import { useDojo } from "./dojo/useDojo";
+import {
+    Dojo_Starter,
+    Direction,
+    MovesEntity,
+    PositionEntity,
+    isPositionEntity,
+    isMovesEntity,
+} from "./dojo/dojo_starter";
 
 function App() {
-    const {
-        setup: {
-            systemCalls: { spawn, move },
-            clientComponents: { Position, Moves },
-        },
-        account,
-    } = useDojo();
+    const dojo_starter = new Dojo_Starter({
+        rpcUrl: "http://localhost:8545",
+        toriiUrl: "http://localhost:5000",
+    });
+
+    const { account } = useDojo();
 
     const [clipboardStatus, setClipboardStatus] = useState({
         message: "",
         isError: false,
     });
 
-    // entity id we are syncing
-    const entityId = getEntityIdFromKeys([
-        BigInt(account?.account.address),
-    ]) as Entity;
-
     // get current component values
-    const position = useComponentValue(Position, entityId);
-    const moves = useComponentValue(Moves, entityId);
+    const position = dojo_starter.position.find({
```

Comment:
How do you imagine compound queries and comparison operators looking here?

Reply:
Quick examples, but they mainly depend on the torii implementation.

```ts
    const moves = dojo_starter.moves.find({
        OR: [
            {
                player: account?.account.address,
            },
            {
                remaining: 0,
            },
        ],
    });
 ```

```ts
    const moves = dojo_starter.moves.find({
        remaining: {
            gte: 0,
        },
    });
```